_sass/_components--quote.scss

```diff
@@ -1,24 +1,43 @@
 $size--quote-border-radius:                         8px;
+$size--quote-padding-x:                             $spacer * 2.25;
+$size--font-weight:                                 600;
+$size--blockquote-padding-y:                        $size--site-section-padding-y;
 
 .quote {
+  background-color: $alabaster;
```

Comment:
@TheMightyPenguin I think this `$alabaster` color may worth to pull it to a variable on `default` file or something like that as it represent the alternative color for sections within the site, so if we change it in some place it will affect to all the sections with that color set as it should be.